api/HTMLMeterElement.json

```diff
@@ -17,7 +17,7 @@
             "version_added": "16"
           },
           "ie": {
-            "version_added": "10"
+            "version_added": false
```

Comment:
https://caniuse.com/meter agrees with these changes, and a quick test of `<meter>` parsing/rendering in IE11 suggests no support indeed.